### PR TITLE
fix(#1316) Disabled keys not passing isDisable prop to item

### LINKED ISSUE #1316: Disabled keys not passing isDisable prop to item
# 🐛 Bug Report 

I've been following the example on the [useMenu](https://react-spectrum.adobe.com/react-aria/useMenu.html) documentation, but I cannot get the `disabledKeys` prop to work properly. The keyboard navigation seems to respect the disabledKeys, but the Item itself receives `undefined` when accessing `item.isDisabled`.

## 🤔 Expected Behavior

The menu item should have the correct `isDisabled` state - either true or false, but not undefined.

## 😯 Current Behavior

When accessing `item.isDisabled`, as written at the documentation, it returns undefined.

## 💻 Code Sample

Here's a Codesandbox to demonstrate the issue:

https://codesandbox.io/s/sweet-bash-ywhx4?file=/src/Menu.js:1041-1067

### PR DESCRIPTION
**Pull request by razvanip**
Closes #1316

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues/1316).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [x] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-aria/menu/docs/useMenu.mdx [hunk 1]
```diff
@@ -128,7 +128,7 @@ import {mergeProps} from '@react-aria/utils';
 
 function Menu(props) {
   // Create state based on the incoming props
-  let state = useTreeState({...props, selectionMode: 'none'});
+  let state = useTreeState({...props, disabledKeys:['one'], selectionMode: 'none'});
```
Comment (LFDanLu) [inline packages/@react-aria/menu/docs/useMenu.mdx:131]: Just a small suggestion: I think this `disabledKeys` should be passed in as one of the Menu props like so
```
<Menu onAction={alert} aria-label="Actions" disabledKeys={["one"]}>
  <Item key="one">One</Item>
  <Item key="two">Two</Item>
  <Item key="three">Three</Item>
</Menu>
```

**Review by LFDanLu — COMMENTED** [review]
Looks good, just one small suggestion, lemme know what you think

File: packages/@react-aria/menu/docs/useMenu.mdx (hunk 1 above)
Comment (razvanip) [reply packages/@react-aria/menu/docs/useMenu.mdx:131]: yeah, better will change it.

File: packages/@react-aria/menu/docs/useMenu.mdx [hunk 2]
```diff
@@ -158,9 +158,11 @@ function Menu(props) {
 function MenuItem({item, state, onAction}) {
   // Get props for the menu item element
   let ref = React.useRef();
+  let isDisabled = state.disabledKeys.has(item.key);
+
   let {menuItemProps} = useMenuItem({
     key: item.key,
-    isDisabled: item.isDisabled,
+    isDisabled: isDisabled,
```
Comment (razvanip) [inline packages/@react-aria/menu/docs/useMenu.mdx:165]: keep `isDisabled: isDisabled` or go for shorthand just `isDisabled` ?

Comment (LFDanLu) [reply packages/@react-aria/menu/docs/useMenu.mdx:165]: shorthand IMO

**Review by LFDanLu — APPROVED** [review]
LGTM, thanks for contributing!

**Review by snowystinger — APPROVED** [review]
lgtm!

**Pull request merged**